ui/src/components/header/Tabs.js

```diff
@@ -47,6 +47,10 @@ class Tabs extends React.Component {
             label: 'History',
             name: 'history',
         },
+        {
```

Comment:
I think we are planning to add features to set user authority filters and expiration later on, in that case having a settings tab would be better instead of tags